site/source/_posts/2019-07-23-whats-new-in-embark-4.1.md

```diff
@@ -0,0 +1,125 @@
+
+- `api start/stop`
+- `webserver start/stop`
+
+The new `service` command lets you start and stop `api`, `webserver`, `blockchain`, `ipfs`, `swarm`, `embark` and `api`:
+
+```
+$ service <service> on/off
+```
+
+This works within Embark's command line Dashboard, as well as [Cockpit's dashboard](https://embark.status.im/docs/cockpit_dashboard.html). To learn more about Embark's interactive console and its command, head over to the [documentation](/docs/using_the_console.html#Enabling-and-disabling-processes).
+
+## Accounts access inside tests
+
+In order to make writing tests in Embark a little bit more convenient, accounts configured and set up via the `config()` function are now injected into `describe()` blocks, making writing tests a little bit more predictable and easier to reason about.
+
+Prior, in order to get hold of accounts within tests, the following was needed:
+
+```
+let accounts = [];
+
+config({
+  contracts: {
+    ...
+  }
+}, (err, _accounts) => {
+  accounts = _accounts;
+});
+
+contract('My contract', () => {
+
+  it('does something', () => {
+    // can use `accounts` here
+  });
+});
+```
+
+Notice that Embark won't run the `contract()` block until `config()` is done doing its work. Therefore, using a global variable was the recommended way to re-initialize `accounts` once `config()` runs its callback.
+
+The same can now be achieved with the following code:
+
+```
+config({
+  contracts: {
+    ...
+  }
+});
+
+contract('My contract', accounts => {
+
+  it('does something', () => {
+    // can use `accounts` here
+  });
+});
+```
+Instead of managing an `accounts` variable yourself, you can just ask for it now within `contract()`'s callback.
+
+## Several improvements inside Cockpit
+
+Cockpit has got a lot of new things as well. This includes [draggable tabs]() inside the code editor, pagination support for [Smart Contracts](https://github.com/embark-framework/embark/commit/d71352b) and the [accounts explorer](https://github.com/embark-framework/embark/commit/745edaf), alphabetically [sorted Smart Contracts](https://github.com/embark-framework/embark/commit/0e9a4a1), and the ability to [send ETH to payable Smart Contract methods](https://github.com/embark-framework/embark/pull/1649) via the Cockpit UI.
+
+
+## What's next?
+
+We've spent a lot of time fixing bugs and revisiting existing, user-facing APIs within Embark and aim to improve those as much as we can to make working with Embark as pleasant as possible. We've been also doing a lot of research and experimentation about integrating with other blockchain platforms, to get Embark ready for the future of decentralization to come.
+
+That's why our next step is to work on v5, where we'll be focussing on making Embark's accounts configuration less confusing and more unified (no more mulitple places to define accounts!), as well as a bunch of internal refactor that may or may not affect existing user applications.
```

Comment:
Don't think saying "that may or may not affect existing user applications." adds much. We can just end it at "a bunch of internal refactor"